apps/web/utils/user/get.ts

```diff
@@ -1,5 +1,47 @@
 import prisma from "@/utils/prisma";
 import type { EmailAccountWithAI } from "@/utils/llms/types";
+import type { Prisma } from "@prisma/client";
+
+export type EmailAccountWithAIAndTokens = Prisma.EmailAccountGetPayload<{
+  select: {
+    id: true;
+    userId: true;
+    email: true;
+    about: true;
+    digestScheduleId: true;
+    digestSchedule: {
+      select: {
+        id: true;
+        intervalDays: true;
+        occurrences: true;
+        daysOfWeek: true;
+        timeOfDay: true;
+        lastOccurrenceAt: true;
+        nextOccurrenceAt: true;
+      };
+    };
+    user: {
+      select: {
+        aiProvider: true;
+        aiModel: true;
+        aiApiKey: true;
+      };
+    };
+    account: {
+      select: {
+        access_token: true;
+        refresh_token: true;
+        expires_at: true;
+      };
+    };
+  };
+}> & {
+  tokens: {
+    access_token: string | null;
+    refresh_token: string | null;
+    expires_at: number | null;
+  };
+};
 
 export async function getEmailAccountWithAi({
```

Comment:
This is used in a few places and the adjustments to the query make it slower due to the join. Most of the time we don't need to fetch digest schedule information. I would create a new function that fetches the data you need rather than editing the existing function that will be used in lots of places.